docs/sources/whatsnew/whats-new-in-v7-4.md

```diff
@@ -125,6 +125,8 @@ Grafana 7.4 includes the following enhancements
 
 > **Note:** We have deprecated browser access mode. It will be removed in a future release.
 
+For more information, refer to the [Elasticsearch docs]({{<relref "../datasources/elasticsearch.md">}}).
```

Comment:
Per the style guide, link text should exactly match the title of the link target whenever possible. Please reconcile.

Reply:
I see you have already made the change. Thank you.

Reply:
I'm not seeing it in GitHub. Weird.